pom.xml

```diff
@@ -126,6 +126,16 @@
       <artifactId>drools-compiler</artifactId>
       <version>${drools.version}</version>
     </dependency>
+    <!--
+    Remove xstream dependency after drools has upgraded its
+    xstream dependency to >= 1.4.14:
+    https://x-stream.github.io/CVE-2020-26217.html
```

Comment:
When I last tried upgrading Drools, I believe it caused an [incompatibility|https://issues.folio.org/browse/CIRC-309] with the Alpine container and so is effectively locked at version 7.0.0, which means we might not be able to upgrade Drools.